depends/common/libssh/11-libssh-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/connect.c
++++ b/src/connect.c
+@@ -476,7 +476,7 @@ int ssh_select(ssh_channel *channels, ssh_channel *outchannels, socket_t maxfd,
+     fd_set *readfds, struct timeval *timeout) {
+   fd_set origfds;
+   socket_t fd;
+-  int i,j;
++  unsigned int i,j;
```

Comment:
Upstream has changed it to `size_t`, https://git.libssh.org/projects/libssh.git/commit/src/connect.c?id=58113d489eecfeb266b82de7bb25d4a0526b9606. Please adjust the patch.
```suggestion
+  size_t i, j;
```